colorbleed/plugins/maya/publish/collect_renderlayers.py

```diff
@@ -126,10 +126,10 @@ def parse_options(self, render_globals):
         # Check for specific pools
         pool_str = attributes.get("pools", None)
         if pool_str:
-            pools = pool_str.split(";")
-            options["renderGlobals"]["Pool"] = pools[0]
-            if len(pools) > 1:
-                options["renderGlobals"]["SecondaryPool"] = pools[1]
+            pool_a, pool_b = pool_str.split(";")
+            options["renderGlobals"].update({"Pool": pool_a})
+            if pool_b and pool_b != "-":
+                options["renderGlobals"].update({"SecondaryPool": pool_a})
```

Comment:
@aardschok BUG! :) `pool_a` -> `pool_b`

Reply:
How about just having it be an empty string, so that it behaves like Deadline does itself? 

That way you can just to `if pool_b`?